docs/reference/pixi_manifest.md

```diff
@@ -36,14 +36,6 @@ The minimally required information in the `project` table is:
 --8<-- "docs/source_files/pixi_tomls/simple_pixi.toml:project"
 ```
 
```

Comment:
`project.name` also become optional? 

https://github.com/prefix-dev/pixi/pull/3514#discussion_r2029926134

Reply:
`project` is an alias for `workspace` these days. But since the docs still reference "project" I changed the text.

Reply:
so now `project` is splited in to `workspace` and `package`?